index.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- <link rel="stylesheet" href="/style/desktopAbout.css"> -->
+  <link rel="stylesheet" href="/style/mobileIndex.css">
+  <link rel="stylesheet" href="/style/DesktopIndex.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous"
+    referrerpolicy="no-referrer" />
+  <title>Alumni Club</title>
+</head>
+
+<body>
+  <div class="top-nav">
+    <div class="top-nav-1">
+        <a href="https://web.facebook.com/chukwuma.mosanya"><i class="fa-solid fa-facebook facebook-icon"></i></a>
+        <a href="https://twitter.com/ChukwumaMosanya"><i class="fa-solid fa-twitter twitter-icon"></i></a>
+    </div>
+
+    <div class="top-nav-2">
+        <a href="#">English</a>
+        <a href="#">My Page</a>
+        <a href="#">Logout</a>
+    </div>
+  </div>
+  <header>
+
+    <nav>
+      <a href="./index.html"></a>
+        <div id="header-logo">
+          <img class="header-logo-image" src="/images/logo.png" alt="logo">
+          <p> GSSO Alumni Ass.</p>
+        </div></a>
+        <div id="nav-items" class="nav-items">
+          <a href="pages/about.html#about">About</a>
+          <a href="#">Program</a>
+          <a href="#">Join</a>
+          <a href="#">Sponsor</a>
+          <a href="#">News</a>
+          <a href="#">CC Campaign</a>
+        </div>
```

Comment:
Kindly fix the styling of these navigation links to resemble the given design 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/191940443-c8a62338-03b8-40df-a9a7-c8f7a6b554e1.png)

Template design: 

![image](https://user-images.githubusercontent.com/73714615/191940411-de702073-ad53-49c3-ab9c-ee6dcef1e9aa.png)